modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,126 @@ Module included in the following assemblies:
+
+This release ends maintenance support for OpenShift {SMProductShortName} version 2.2. If you are using OpenShift {SMProductShortName} version 2.2, you should upgrate to a supported version.
+
+=== Component versions for {SMProductName} version 2.5
+
+|===
+|Component |Version
+
+|Istio
+|1.18.5
+
+|Envoy Proxy
+|1.26.8
+
+|Kiali
+|1.73.4
+|===
+
+=== Istio 1.18 support
+
+Service Mesh 2.5 is based on Istio 1.18, which brings in new features and product enhancements. While {SMProductName} supports many Istio 1.18 features, the following exceptions should be noted:
+
+* Ambient mesh is not supported
+* QuickAssist Technology (QAT) PrivateKeyProvider in Istio is not supported
+
+=== Cluster-Wide mesh migration
+
+This release adds documentation for migrating from a multitenant mesh to a cluster-wide mesh. For more information, see the following documentation:
+
+* About migrating to a cluster-wide mesh
+* Excluding namespaces from a cluster-wide mesh
+* Defining which namespaces receive sidecar injection in a cluster-wide mesh
+* Excluding individual pods from a cluster-wide mesh
+
+=== {SMProductName} operator on ARM-based clusters
+
+This release provides the {SMProductName} Operator available on ARM-based clusters as a generally available feature. 
+
+=== Integration with {DTProductName} (Tempo) Stack
+
+This release introduces a generally available integration of the tracing extension provider(s). You can expose tracing data to the {DTProductName} (Tempo) stack by appending a named element and `zipkin` provider to the `spec.meshConfig.extensionProviders` specification. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
+
+[NOTE]
+====
+{DTProductName} (Tempo) Stack is not supported on {ibm-z-title}. 
+====
+
+=== OpenShift Service Mesh Console plugin
+
+This release introduces a generally available version of the OpenShift {SMProductShortName} Console (OSSMC) plugin. 
+
+The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option is available in the left-hand navigation of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages. 
+
+The features of the OSSMC plugin are very similar to those of the standalone Kiali Console. The OSSMC plugin does not replace the Kiali Console, and after installing the OSSMC plugin, you can still access the standalone Kiali Console. 
+
+=== Istio OpenShift Routing (IOR) default setting change
+
+The default setting for Istio OpenShift Routing (IOR) has changed. Starting with this release, automatic routes are disabled by default for new instances of the `ServiceMeshControlPlane` resource.
```

Comment:
Should we add a sentence for what this means for existing `ServiceMeshControlPlane` resources?

Reply:
The word "new" hints that as an existing user, I shouldn't worry about this. I am concerned that people might miss this word and worry that it may cause IOR to disable for their existing mesh and for that to break their existing Gateways. Thus, I wonder if to be super clear, we should add a sentence like "This will not impact existing `ServiceMeshControlPlane` instances, which will continue to have IOR enabled by default."

Perhaps another question though - for engineering... if I update an existing `ServiceMeshControlPlane` instance, will the setting continue to default to enabled? (Just want to double check).

Reply:
Thanks for clarifying for me. I'll make an update.

Reply:
Done. Added sentence we vetted in slack.